src/hooks/common.ts

```diff
@@ -29,7 +29,7 @@ export const resolveDeviceType = (
 			}
 
 			return api.get({
-				resource: 'device_type',
+				resource: 'device_type_table',
```

Comment:
Is there a small window of time after the API starts and until device types are populated where this might return no results?

Reply:
I think it might.